operations/network/external_dns/Ingress_Routing.md

```diff
@@ -1,27 +1,27 @@
 # Ingress Routing
 
-Ingress routing to services via Istio's ingress gateway is configured by VirtualService custom resource definitions \(CRD\). When using external hostnames, there needs to be a VirtualService CRD that matches the external hostname to the desired destination.
+Ingress routing to services via Istio's ingress gateway is configured by `VirtualService` custom resource definitions \(CRD\). When using external hostnames, there needs to be a `VirtualService` CRD that matches the external hostname to the desired destination.
 
-For example, the configuration below controls the ingress routing for prometheus.cmn.SYSTEM_DOMAIN_NAME:
+For example, the configuration below controls the ingress routing for `prometheus.cmn.SYSTEM_DOMAIN_NAME`:
 
 ```bash
 ncn-w001# kubectl get vs -n sysmgmt-health cray-sysmgmt-health-prometheus
```

Comment:
This might be extra work, if so just ignore/dismiss. Could you please also fix-up the prompts in this page to be more copy-paste friendly? This should move the `ncn-w001#` prompt out of all the code snippets so that this is more copy-paste friendly. https://github.com/Cray-HPE/docs-csm/blob/MTL-1695/introduction/documentation_conventions.md#step-examples

Reply:
I need to keep the scope of this ticket limited to the changes required for the ticket and the changes required to pass the lint checkers.

Reply:
@rustydb you already made this change in your MTL-1695 PR.   That has now been rebased in.